proxy/InkAPI.cc

```diff
@@ -9112,6 +9113,66 @@ TSSslContextDestroy(TSSslContext ctx)
   SSLReleaseContext(reinterpret_cast<SSL_CTX *>(ctx));
 }
 
+void
+TSRegisterProtocolSet(TSVConn sslp, TSNextProtocolSet ps)
+{
+  NetVConnection *vc        = reinterpret_cast<NetVConnection *>(sslp);
+  SSLNetVConnection *ssl_vc = dynamic_cast<SSLNetVConnection *>(vc);
+  if (ssl_vc) {
+    ssl_vc->clearnpnSet();
```

Comment:
clearnpnset is freeing the already created protoscolstring to avoid memory leak. I need to change the function name to make that meaningful